src/components/index.d.ts

```diff
@@ -11,6 +11,15 @@ declare class UniversalNavbar extends React.Component {
 }
 export { UniversalNavbar }
 
+declare class UniversalNavbarExpanded extends React.Component {
+  state: {
+    showMobileMenu: boolean
+  }
+  toggleHamburger: () => void
+  render(): JSX.Element
+}
+export { UniversalNavbarExpanded }
+
```

Comment:
These types are not correct. 
This isn't blocking because we're not importing `UniversalNavbarExpanded` from a Typescript file.
You can see more in this PR: https://github.com/getethos/ethos-design-system/pull/275/files

Reply:
noted - was the old one also incorrect for `UniversalNavbar`?

Reply:
Yes

Reply:
marked in TODOs for fastfollow